variables/_svgs.pcss

```diff
@@ -106,6 +106,25 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Close
+ * ----------------------------------------------------------------------------- */
+
+@svg-load close url('../icons/close.svg') {
+
+	path {
+		stroke: var(--color-icon-secondary);
+	}
+}
+
+@svg-load close-focus url('../icons/close.svg') {
```

Comment:
let's call this `close` and above `close-secondary`, to follow the icon color we're using (see other uses in this file).